homeassistant/components/shelly/config_flow.py

```diff
@@ -37,6 +43,13 @@
 HOST_SCHEMA: Final = vol.Schema({vol.Required(CONF_HOST): str})
 
 
+BLE_SCANNER_OPTIONS = [
+    selector.SelectOptionDict(value=BLEScannerMode.DISABLED, label="Disabled"),
+    selector.SelectOptionDict(value=BLEScannerMode.ACTIVE, label="Active"),
```

Comment:
I wish we could make it more clear what it means to be active/passive.

Reply:
<img width="574" alt="Screenshot 2022-11-15 at 7 49 09 AM" src="https://user-images.githubusercontent.com/663432/201935831-203465b2-b6b6-48ba-a8a0-6e4834209c91.png">

Added a description